python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -187,7 +187,10 @@ def parent_newer() -> "ParentNewerCondition":
         return ParentNewerCondition()
 
     @staticmethod
-    def eager() -> "SchedulingCondition":
+    def eager(
```

Comment:
Hmmm I am a little concerned at assigning a default time like this in a public API as it seems very arbitrary.

What evidence do we have that an hour is the right value and that users will expect this behavior?